092-self-healing-using-cruise-control.md

```diff
@@ -0,0 +1,253 @@
+# Self Healing using Cruise Control
+
+This proposal is about enabling the self-healing properties of Cruise Control in the Strimzi operator.
+The self-healing feature of Cruise Control allows us to heal the anomalies detected in the cluster, automatically. 
+Anomalies are detected by Cruise Control using the anomaly detector manager.
+
+## Current situation
+
+During normal operations it's common for Kafka clusters to become unbalanced (through partition key skew, uneven partition placement etc.) or for problems to occur with disks or other underlying hardware which makes the cluster unhealthy.  
+Currently, if we encounter any such scenario we need to fix these issues manually i.e. if there is some goal violation and the cluster is imbalanced then we might move the partition replicas across the brokers to fix the violated goal by using the `KafkaRebalance` custom resource in the [`rebalance`](https://strimzi.io/docs/operators/latest/full/deploying.html#proc-generating-optimization-proposals-str) mode.
```

Comment:
There is no `rebalance` mode. Did you mean `full`? Anyway I would not specify it, because it's possible you could even decide to scale up the cluster when doing the rebalancing so `add-brokers` would be needed.